components/WhyMeditate/WhyMeditate.module.css

```
@@ -0,0 +1,110 @@
+.contentContainer {
+  min-height: 100%;
+  background: url(/buddha-bg2.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+}
+.title {
+  margin: 0;
+  padding: 2% 4%;
+}
+.reason {
+  max-width: 620px;
+  height: 48px;
+  padding: 0 4%;
+}
+.audioContainer {
+  background-color: #fffffffa;
+  border-radius: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-clip: content-box;
+  overflow: hidden;
+}
+
+.audioContainer.isPlayed {
+  left: 0;
+  top: 0;
+
+  border: 15px solid rgba(0, 0, 0, 0);
+}
+
+.playerContainer {
+  position: relative;
+  margin-left: 5%;
+  width: 70px;
+  height: 70px;
+  transform-origin: top;
+  transition: all 1.2s ease;
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-family: 'Libre Baskerville', serif;
+}
+.playerContainer.isPlayed {
+  margin-top: 0;
+  width: 600px;
+  height: 600px;
+  max-width: calc(100% - 20px);
+}
+
+@keyframes appear {
+  100% {
+    opacity: 1;
+  }
+}
+
+.progress {
+  fill: none;
+  stroke-width: 4;
+  stroke: #a85a5d;
+  transform: rotate(270deg);
+  transform-origin: center center;
+  stroke-dasharray: 760;
+  stroke-dashoffset: 760;
+  stroke-dashoffset: calc(756 - (756 * var(--currentProgress)));
+}
+
+.audio {
+  margin-top: 10px;
+}
+.progress {
+  stroke-linecap: round;
+}
+.playPauseButton {
+  animation: appear 100ms ease-in forwards;
+  opacity: 0;
+}
+.speedButton {
+  border-radius: 50%;
+  border: none;
+  min-height: 50px;
+  min-width: 50px;
+  font-size: 1.5em;
+  font-style: italic;
+  color: #a85a5d;
+  animation: delay 1.3s ease-out;
+}
```

I think it would look nicer like this: 

```suggestion
.speedButton {
   top: -80px;
    border-radius: 50%;
    border: none;
    min-height: 50px;
    background: none;
    min-width: 50px;
    font-size: 1.2em;
    font-style: italic;
    color: #a85a5d;
    position: relative;
}
```